crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/prosecution_concluded/controller/ProsecutionConcludedControllerTest.java

```diff
@@ -38,27 +35,24 @@ class ProsecutionConcludedControllerTest {
     @MockBean
     private TraceIdHandler traceIdHandler;
 
-    @MockBean
-    private DeadLetterMessageService deadLetterMessageService;
-
     @Test
     void givenIncorrectParameters_whenGetCountByMaatIdAndStatusIsInvoked_thenErrorIsThrown()
             throws Exception {
-        mvc.perform(MockMvcRequestBuilders.head(COUNT_ENDPOINT_URL)
+        mvc.perform(MockMvcRequestBuilders.get(String.format(COUNT_ENDPOINT_URL, "invalid"))
                         .contentType(MediaType.APPLICATION_JSON))
-                .andExpect(status().isNotFound());
+                .andExpect(status().isBadRequest());
     }
 
     @Test
-    void givenAValidParameters_whenGetCountByMaatIdAndStatusIsInvoked_thenReturnCount()
+    void givenValidParameters_whenGetCountByMaatIdAndStatusIsInvoked_thenReturnCount()
             throws Exception {
         when(service.getCountByMaatIdAndStatus(TestModelDataBuilder.TEST_REP_ID,
                 CaseConclusionStatus.PENDING.name())).thenReturn(1L);
-        mvc.perform(MockMvcRequestBuilders.head(
+        mvc.perform(MockMvcRequestBuilders.get(
                         String.format(COUNT_ENDPOINT_URL, TestModelDataBuilder.TEST_REP_ID)
                                 + "?status=" + CaseConclusionStatus.PENDING.name()))
                 .andExpect(status().isOk())
-                .andExpect(content().string(""))
-                .andExpect(header().string(HttpHeaders.CONTENT_LENGTH, "1"));
+                .andExpect(content().string(String.valueOf(1)))
+                .andExpect(content().contentType(MediaType.APPLICATION_JSON));
```

Comment:
To complement the previous comment, this is why I believe we are returning invalid JSON.